Dockerfile-default-cpu

```diff
@@ -9,9 +9,9 @@ ARG TARGETPLATFORM
 ARG TENSORFLOW_PIP
 ARG TORCH_PIP
 ARG TORCHVISION_PIP
-ARG TORCH_PROFILER_GIT
 RUN if [ "$TENSORFLOW_PIP" ]; then \
-        if [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
+        conda install h5py \
+        && if [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
             TENSORFLOW_AARCH64_PIP=$(printf '%s' "$TENSORFLOW_PIP" | sed 's/cpu/aarch64/') \
```

Comment:
Let's replace conda with `pip install h5py==3.7.0` ?
Also let's move this install inside the AARCH64 clause because I think that is the only architecture that has a problem with h5py.

Reply:
Actually, now it looks like they belatedly built the binary for aarch64 and uploaded it to [PyPI](https://pypi.org/project/h5py/#files).

<img width="727" alt="image" src="https://user-images.githubusercontent.com/17324129/214694410-c47a66c2-8f62-4f39-b269-1664b4f27250.png">